src/api/app/views/webui/users/index.html.haml

```diff
@@ -3,7 +3,11 @@
 .card.mb-3
   = render partial: 'webui/configuration/tabs'
   .card-body
-    %h3= @pagetitle
+    %h3
+      = @pagetitle
+      - if feature_enabled?(:responsive_ux)
+        = link_to(new_user_path(pagetitle: 'create user', submit_btn_text: 'create'), title: 'Create User', id: 'create-user') do
```

Comment:
The `id` isn't needed, so we could remove it.

```suggestion
        = link_to(new_user_path(pagetitle: 'create user', submit_btn_text: 'create'), title: 'Create User') do
```

Reply:
Done